src/PokemonGym.java

```diff
@@ -0,0 +1,305 @@
+import java.util.*;
+
+ interface PokemonGym {
```

Comment:
Het is niet de bedoeling dat je methodes bodies hebben in een interface. In een interface definieer je de methodes zonder de accolades. In de implementatie van een interface kun je dan de body (alles tussen de accolades) invullen.